docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -43,7 +49,7 @@ parent:
     uri: https://raw.githubusercontent.com/eclipse/che-devfile-registry/master/devfiles/nodejs/devfile.yaml
```

Comment:
The link is a devfile 1.0 file. Suggest to change to use an example that points to a devfile 2.0 file, e.g. https://raw.githubusercontent.com/devfile/registry/master/stacks/nodejs/devfile.yaml